### DISCUSSION TITLE
Library update not completing 20231216

### DISCUSSION THREAD

**Discussion by RamblerKevin** [rern/rAudio#1542] [General]
Hi,

I updated to a brand new install of 20231216 over the weekend (64bit for Raspberry Pi 4) and the library update for my USB drive is never completing. It just keep flashing with the library update (6 hours and counting) and the setting menu icon next to library is constantly blue. The previous version I had didn't have any issues with the same drive. 

I rebooted, ejected and had the software forget the drive and plugged it back in with a fresh library update and it's doing the same thing.

The drive has 1.4 gb of music files. I messed around with the 'buffer output' setting but hasn't seemed to change this. 

Music still plays but when I select that drive but it flashes when you select directories and isn't as quick as it was on my older version, since it still seems like its scanning.

Any suggestions?

Thanks

Edit: I did get it to stop by clicking the library refresh icon and prompting the 'update in process' 'stop?' command but it still seems to be lagging a bit and I haven't checked to see if all the files loaded.

**Comment by cmh714** [comment]
try using the Only Changed Files selection. GL!
